reflex-dom-core/src/Reflex/Dom/Builder/Immediate.hs

```diff
@@ -1170,7 +1171,7 @@ textNodeInternal tc@(TextNodeConfig !t mSetContents) = do
     HydrationMode_Hydrating -> addHydrationStepWithSetup (maybe (pure $ pure t) (hold t) mSetContents) $ \currentText -> do
       n <- hydrateTextNode doc =<< sample currentText
       mapM_ (requestDomAction_ . fmap (setNodeValue n . Just)) mSetContents
-  pure $ TextNode ()
+  pure $ TextNode () never
```

Comment:
This should not be `never`: it should match the immediate version after hydration switchover